migrations/2024_10_16_142826_create_pets_table.php

```diff
@@ -6,23 +6,26 @@
 
 return new class extends Migration
 {
-    /**
-     * Run the migrations.
-     */
+
     public function up(): void
     {
         Schema::create('pets', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->string('nome');
             $table->date('data_nascimento');
             $table->timestamps();
+//            $table->unsignedBigInteger('especie_id');
+//            $table->foreign('especie_id')
+//                ->references('id')
+//                ->on('especie')
+//                ->onDelete('cascade');
+
+//            $table->foreign('especies_id')
+
```

Comment:
Evitar código comentado em PR

Reply:
tirei, no caso codigo comentado só se for algo que eu vá usar mais pra frente? @th4deu